### PR TITLE
[issues] category dropdown scroll initial attempt

### PR DESCRIPTION
**Pull request by philli-m**

### CODE REVIEW

**Comment by philli-m** [comment]
In case anyone gets onto this before I get back: I was hoping we could just add a config but we do so much nesting I think we may have to refactor to simplify our form screens, did not extensively play around with configs though.

File: src/components/formFields.js [hunk 1]
```diff
@@ -68,20 +64,12 @@ export const CheckBoxFormField = (props) => {
...
 
-export const DropdownFormFieldContainer = (props) => {
-  return (
-    <View style={styles.dropdownContainer}>
-      <FormLabel>{props.field}</FormLabel>
-      {props.children}
-    </View>
-  )
-}
-
```
Comment (philli-m) [inline src/components/formFields.js:79]: seems to be pointless as dropdown will always need a label

**Comment by philli-m** [comment]
Guy released a beta fix last week :D assume it's ok to use beta till the release anyway right?

**Comment by khamui** [comment]
sweet! nested scroll works for both OSs.

Note: I had to start with clearing cache!

**Pull request merged**